Framework/Algorithms/src/MergeRuns/SampleLogsBehaviour.cpp

```diff
@@ -396,19 +458,46 @@ void SampleLogsBehaviour::checkErrorProperty(
     const MatrixWorkspace_sptr &addeeWS, Property *addeeWSProperty,
     const SampleLogBehaviour &behaviour, const double addeeWSNumber,
     const double outWSNumber, const std::string name) {
+
+  if (!isWithinTolerance(behaviour, addeeWSNumber, outWSNumber) &&
+      !stringPropertiesMatch(behaviour, addeeWSProperty)) {
```

Comment:
This is now slightly different from before? Previously the string property check was not taken into account in the "numeric" code branch?